scalameta/dialects/shared/src/main/scala/scala/meta/dialects/package.scala

```diff
@@ -103,7 +103,7 @@ package object dialects {
   implicit val ParadiseTypelevel212 = Typelevel212
     .withAllowInlineMods(true)
 
-  implicit val Scala3 = Scala213
```

Comment:
Should it maybe stay as a alias for the latest Scala 3 dialect?

Reply:
it is, please see below